easy_my_coop_website_portal/views/easy_my_coop_website_portal_templates.xml

```diff
@@ -0,0 +1,203 @@
+     Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<openerp>
+
+  <!-- Add cooperator information -->
+  <template
+    id="website_portal_details_form"
+    name="Website Portal Details Form"
+    inherit_id="website_portal_v10.portal_layout">
+    <xpath expr="//div[@class='o_my_details']" position="after">
+      <div class="o_my_details_coop" t-if="coop.member">
+        <h3 class="page-header">Your Cooperator Details</h3>
+        <p class="text-center">
+          <span t-if="coop.member">
+            You are an effective cooperator
+          </span>
+          <span t-if="not coop.member">
+            You are not a cooperator
+          </span>
+        </p>
+        <p t-if="coop.cooperator_register_number">
+          <label>Cooperator Number: </label>
+          <t t-esc="coop.cooperator_register_number"/>
+        </p>
+        <p t-if="coop.effective_date">
+          <label>Cooperator Entrance Date: </label>
+          <span t-field="coop.effective_date"/>
+        </p>
+        <p t-if="coop.number_of_share">
+          <label>Number of Share: </label>
+          <t t-esc="coop.number_of_share"/>
+        </p>
+        <p t-if="coop.share_ids">
+          <div t-foreach="coop.share_ids" t-as="share">
+            <span t-field="share.effective_date"/>:
+            <t t-esc="share.share_number"/> x
+            <t t-esc="share.share_short_name"/>
+            (<span t-field="share.share_unit_price"
+              t-field-options='{
+                "widget": "monetary",
+                "display_currency": "coop.company_id.currency_id"
+              }'/>) -
+            <span t-field="share.total_amount_line"
+              t-field-options='{
+                "widget": "monetary",
+                "display_currency": "coop.company_id.currency_id"
+              }'/>
+          </div>
+        </p>
+        <p t-if="coop.total_value">
+          <label>Total Value of Share: </label>
+          <span t-field="coop.total_value"
+            t-field-options='{
+              "widget": "monetary",
+              "display_currency": "coop.company_id.currency_id"
+            }'/>
+        </p>
+
+        <p t-if="coop.gender">
+          <label>Gender: </label>
+          <t t-esc="coop.gender"/>
+        </p>
+        <p t-if="coop.birthdate">
+          <label>Date of Birth: </label>
+          <t t-esc="coop.birthdate"/>
+        </p>
+        <p t-if="coop_bank and coop_bank.acc_number">
+          <label>Bank Account: </label>
+          <t t-esc="coop_bank.acc_number"/>
+        </p>
+        <p t-if="coop.national_register_number">
+          <label>National Register Number: </label>
+          <t t-esc="coop.national_register_number"/>
+        </p>
+        <p t-if="coop.company_register_number">
+          <label>Company Register Number: </label>
+          <t t-esc="coop.company_register_number"/>
+        </p>
```

Comment:
this should be displayed only if coop.is_company

Reply:
You're right.